### ISSUE TITLE
OnQueryError isn't getting called for query-errors at all

### ISSUE THREAD

**Issue by dotSILENT** [pBlueG/SA-MP-MySQL#204]
OnQueryError callback is only being called for connection errors, for example trying to connect to an offline local database, which results in an error 2002 (CR_CONNECTION_ERROR) that isn't even defined in the include file.
At least on Windows 10 1803, no query errors (tested on syntax and non-existing tables) are triggering the On**Query**Error. And I'm not using the ORM. Errors are being logged to the mysql.log file, but the callback isn't getting called.
Haven't checked it on Linux yet, but I'm assuming it may be the same.
Detecting query-specific errors is kinda useful, for example when you don't want to corrupt/loose the data of the players. I normally crashed the server in case of any error in StrickenKid's plugin, would like to do the same here.

**Comment by maddinat0r** [comment]
Yeah, `OnQueryError` gets only called for `mysql_tquery` and `mysql_pquery`. 
I'll look into this. I'm currently not sure if I'll add a call to `OnQueryError`, because you can use `mysql_errno` and `mysql_error` to check for errors after an unthreaded query. However, having a central callback for all query errors (including ORM-generated queries, as they currently also do not call any error callback) would make it easier to use for everyone.
